### PR TITLE
Fix tiva port

### PR DESCRIPTION
**Pull request by utzig**
The Tiva port was updated to be compatible with new OS structure and naming schemes. All demos are already tested.

I will merge by the end of the week!

### CODE REVIEW

**Comment by utzig** [comment]
@marcoveeneman If you have some time please double check this.

File: demos/TIVA/RT-TM4C1294-LAUNCHPAD-LWIP/web/web.h [hunk 1]
```diff
@@ -41,7 +41,7 @@ extern THD_WORKING_AREA(wa_http_server, WEB_THREAD_STACK_SIZE);
 #ifdef __cplusplus
 extern "C" {
 #endif
-  msg_t http_server(void *p);
+  void http_server(void *p);
```
Comment (marcoveeneman) [inline demos/TIVA/RT-TM4C1294-LAUNCHPAD-LWIP/web/web.h]: This could be `THD_FUNCTION(http_server, p);`

**Comment by marcoveeneman** [comment]
Hi Fabio,

I already noticed some things changed in the main repo so this was already on my todo list, thanks for the patch. I just had a single comment about the declaration of http_server in web.h ;) Other than that everything looks fine to me.

**Comment by utzig** [comment]
Marco, I fixed the issue. Thanks for the review!

**Pull request merged**